src/main/java/com/marginallyclever/makelangelo/turtle/turtleRenderer/TurtleRenderFactory.java

```diff
@@ -11,18 +12,27 @@
  * @author Dan Royer
  */
 public enum TurtleRenderFactory {
-	DEFAULT("Default", new DefaultTurtleRenderer()),
-	BARBER_POLE("Barber pole", new BarberPoleTurtleRenderer()),
-	SEPARATE_LOOP("Separate loops",new SeparateLoopTurtleRenderer()),
-	MARLIN_SIM("Marlin simulation",new MarlinSimulationVisualizer());
+	DEFAULT("Default", new DefaultTurtleRenderer(),Translator.get("TurtleRenderFactory.DEFAULT")),// not coder friendly but allow latter the CI test not to miss this traduction keys.
+	BARBER_POLE("Barber pole", new BarberPoleTurtleRenderer(),Translator.get("TurtleRenderFactory.BARBER_POLE")),
+	SEPARATE_LOOP("Separate loops",new SeparateLoopTurtleRenderer(),Translator.get("TurtleRenderFactory.SEPARATE_LOOP")),
+	MARLIN_SIM("Marlin simulation",new MarlinSimulationVisualizer(),Translator.get("TurtleRenderFactory.MARLIN_SIM"));
 
 	private final TurtleRenderer turtleRenderer;
 
 	private final String name;
-
-	TurtleRenderFactory(String name, TurtleRenderer turtleRenderer) {
+	
+	private final String traductedText;
+
+	/**
+	 * 
+	 * @param name
+	 * @param turtleRenderer
+	 * @param traductedText the text (traducted) N.B. : not base on the name (used in findByName) to avoid a bad traduction (that give for two elements the same value) to assert any traduction may not create issues... 
```

Comment:
the word "traduction" does not exist in english, you should use "translation" instead

Reply:
Thanks ! and my apologies for my poor level in English.